### DISCUSSION TITLE
[FR] <title> Status code: 403

### DISCUSSION THREAD

**Discussion by psalmuel19** [cjbarrie/academictwitteR#254] [Q&A]
### Describe the solution you'd like

Hi, my script was interrupted before it finished scraping. I tried to see if I could resume collection with resume_collection(data_path = “data”) but it didn't work. 
Then I thought of maybe just running with what we have but the bind_tweets function isn't working (strangely). 
Now trying to start afresh, the same code from yesterday is returning an error: make_query(url = endpoint_url, params = params, bearer_token = bearer_token,

**Comment by psalmuel19** [comment]
Fixed the error by restarting after some minutes. But the resume_collection not working

**Comment by monica-weedman** [comment]
I also ran into this issue several times yesterday. Below is the exact error I received (python):

Traceback (most recent call last):
  File "/Users/main.py", line 130, in <module>
    json_response = connect_to_endpoint(url[0], headers, url[1], next_token)
  File "/Users/main.py", line 43, in connect_to_endpoint
    raise Exception(response.status_code, response.text)
Exception: (403, '{"client_id":"20695550","detail":"When authenticating requests to the Twitter API v2 endpoints, you must use keys and tokens from a Twitter developer App that is attached to a Project. You can create a project via the developer portal.","registration_url":"https://developer.twitter.com/en/docs/projects/overview","title":"Client Forbidden","required_enrollment":"Standard Basic","reason":"client-not-enrolled","type":"https://api.twitter.com/2/problems/client-forbidden"}')
Endpoint Response Code:403

Process finished with exit code 1
